client/src/components/MapViewEntry.jsx

```diff
@@ -11,40 +12,44 @@ type Props = {
 };
 
 const MapViewEntry = (props: Props) => {
-  const { selected } = props;
+  const { resource, selected } = props;
   const color = selected ? '#1890FF' : '#D9D9D9';
 
   return (
     <>
       <Row className="entry">
         <Col span={16}>
           <Row>
-            <b>Crisis Nursery</b>
+            <b>{resource.name}</b>
           </Row>
           <Row className="row">
-            <Col>Urbana</Col>
-            <Col className="second">•</Col>
-            <Col className="third">1309 West Hill Street</Col>
+            <Col>{resource.city}</Col>
+            <Col className="second-element">•</Col>
+            <Col className="third-element">
+              {resource.address || 'Address not provided'}
+            </Col>
           </Row>
           <Row>
             <Col>$</Col>
-            <Col className="second">•</Col>
-            <Col className="third">English, Spanish, French, Chinese</Col>
+            <Col className="second-element">•</Col>
+            <Col className="third-element">
+              {resource.languages.join(', ') || 'English'}
```

Comment:
I'm doing some super jank thing to show the languages in their respective language, can you do this here too 😂 https://github.com/hack4impact-uiuc/ymca/blob/0369e54d2fd3ffd72fd516b7fd65a5e391229abe/client/src/components/ResourcePreview.jsx#L72